line-bot-model/src/main/java/com/linecorp/bot/model/response/GetFriendsDemographicsResponse.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.bot.model.response;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import com.linecorp.bot.model.response.GetFriendsDemographicsResponse.GetFriendsDemographicsResponseBuilder;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+@JsonDeserialize(builder = GetFriendsDemographicsResponseBuilder.class)
+public class GetFriendsDemographicsResponse {
+
+    @Value
+    @Builder
+    @JsonDeserialize(builder = GenderWisePercentage.GenderWisePercentageBuilder.class)
+    static class GenderWisePercentage {
+        String gender;
+        Double percentage;
```

Comment:
```suggestion
        double percentage;
```

if it is never null.

Reply:
And please add comment to `[0.0-100.0]`.